### ISSUE TITLE
ready for action ?

### ISSUE THREAD

**Issue by ghost** [rimusz/coreos-multi-node-k8s-gce#4]
i noticed there were no releases yet for this. Is this ready to give it a go.
We woudl not be using for production yet, but giving it lots of work and testing and monitoring.

i am using the OSX tools you wrote that are excellent, but now i want to deploy to GCE.

**Comment by rimusz** [comment]
yep, this one works too. I never put to master non working version. :-)

And I'm working on much better scripts version to provision Kubernetes to GCE, but I'm 
a bit busy with different things but will kick this one development over next weekend as I
want to have very easy deployment of CoreOS+Kubernetes production clusters to GCE myself.

**Comment by ghost** [comment]
ok thanks, i will give it a kicking later this week. 

i need some advice... How best to develop Micro Services.
a. i can develop docker images
b. I can push to our own docker repo
c. But the k8 you have done does not support docker. It only seems to support bare metal on top of docker. 

Am i missing something ?

**Comment by rimusz** [comment]
Usually I work on docker images with my single cores VM app,
then I push images to our private docker repo

The k8s script/App works with CoreOS + K8s cluster and k8s for now only works with docker, it looks like later one it will support CoreOS Rocket too.

So you easily use your docker images with k8s script on GCE.

**Comment by ghost** [comment]
ok  - that sounds really promising.  i gotta work on that path then. 

So first step.
Can you point me to a dockerised Docker repo ? I also saw a Docker repo GUI.
I then assume i wil have to change the path to the docker repo server on OSX. So when i download an update it will get stomped away. any advice there ?

**Comment by rimusz** [comment]
as the private docker registry I use https://registry.hub.docker.com/_/registry/ with the local storage for repos.
I tried https://registry.hub.docker.com/u/google/docker-registry/ with GC storage but I found it a bit slow too push/pull images for my needs.

As registry UI, I like this one https://registry.hub.docker.com/u/konradkleine/docker-registry-frontend/

I do not really understand what you mean here:
I then assume i wil have to change the path to the docker repo server on OSX. So when i download an update it will get stomped away. any advice there ?
